src/olympia/devhub/tests/test_views.py

```diff
@@ -2261,6 +2265,26 @@ def test_desktop_excludes_mobile(self):
         self.check_excluded_platforms('desktop.xpi', [
             str(p) for p in amo.MOBILE_PLATFORMS])
 
+    def test_webextension_supports_all_platforms(self):
+        self.create_appversion('firefox', '*')
+        self.create_appversion('firefox', '42.0')
+        # Android is only supported 48+
+        self.create_appversion('android', '48.*')
+
+        self.check_excluded_platforms('valid_webextension.xpi', [])
```

Comment:
This test is failing, it's returning `[ANDROID.id]`. Seems bad to me, but I don't see why yet.